Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,45 @@
 ﻿using System;
+using System.Runtime.Intrinsics.X86;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello first user! What's your name?");
+            string name1 = Console.ReadLine();
+            Console.WriteLine("Great, and what's your surname?");
+            string surname1 = Console.ReadLine();
+            Console.WriteLine("Amazing, and how old are you?");
+            string age1 = Console.ReadLine();
+            Console.WriteLine("Cool, and what's your weight?");
+            string weight1 = Console.ReadLine();
+            float w1 = float.Parse(weight1);
+            Console.WriteLine("Groovie, and how tall are you?");
+            string height1 = Console.ReadLine();
+            float h1 = float.Parse(height1);
+            float bmi1 = (w1 / h1 / h1) * 10000;
```

Comment:
You could separate the bmi from the rest as well, because it's quite different from the others.